neard/Cargo.toml

```diff
@@ -23,6 +23,7 @@ borsh = "0.7.0"
 tracing = "0.1.13"
 tracing-subscriber = "0.2.4"
 num-rational = { version = "0.2.4", features = ["serde"] }
+curl = "0.4.30"
```

Comment:
We already had `actix-web` (it is a server and a client) in our dependencies, we should have preferred using existing Rust dependencies instead of bringing new C dependencies (curl brings libcurl and who-knows-what openssl version)

Reply:
Yes, I tried to get it working with actix-web from the start, had some troubles with timeouts while downloading the genesis due to being large ~150Mb, I landed this before the comment, so currently yes libcurl would have to be installed. I can try to get it working with this a bit more, but didn't have luck first time. Felt like I spent too much time on it and decided to use curl instead.